src/main/java/org/jabref/logic/externalfiles/LinkedFileHandler.java

```diff
@@ -59,6 +59,10 @@ public boolean moveToDefaultDirectory() throws IOException {
                     filePreferences.getFileDirectoryPattern());
         }
 
+        if (filePreferences.getFileDirectory().isPresent()) {
+            targetDirectoryName = filePreferences.getFileDirectory().get().toString() + "\\" + targetDirectoryName;
```

Comment:
`\\` is a Windows path separator.

Reply:
Thank you for raising this critical question. This time I used File.separator to ensure the separator of different systems.